Gemfile

```diff
@@ -13,22 +13,22 @@ gem 'jquery-rails'
 gem 'pg'
 
 group :development do
-  gem 'thin'
-  gem "better_errors"
-  gem "binding_of_caller"
+  gem 'thin', '~> 1.6.3'
+  gem 'better_errors'
+  gem 'binding_of_caller'
+  gem 'letter_opener'
 end
 
 group :development, :test do
   gem 'pry'
   gem 'pry-nav'
-  # gem 'pry-rails'
 end
 
 group :test do
   gem 'fabrication'
   gem 'faker'
   gem 'rspec-rails', '2.99'
-  gem 'database_cleaner', '1.2.0'
+  gem 'database_cleaner'
```

Comment:
You still have `database_cleaner (1.2.0)` in your `Gemfile.lock` - I recommend upgrading to `1.4.1`.